pages/development/defining-components.mdx

```diff
@@ -181,3 +181,46 @@ export default defineComponent(Component, meta, {
 ```
 
 The `defineComponent` and `loadData` functions used above are two of the **most powerful** parts of the Embeddable SDK. You can learn more about them here: [defineComponent](./define-component-function) and [loadData](/development/loading-data).
+
+## Extending Vanilla Components
+
+You can also **extend existing components** like the Bar Chart or Pivot Table to add custom functionality. This allows you to leverage the existing UI while adding your own logic or data handling. There are two ways to do this:
```

Comment:
@cwbuecheler given there's a chance that the reader might not know what Vanilla components are, when they're reading this page, can I suggest you make "existing components" a link like this: [existing components](/dashboards/starter-components)